quodlibet/po/fr.po

```diff
@@ -2652,18 +2644,16 @@ msgid ""
 msgstr ""
 
 #: ../quodlibet/ext/gstreamer/karaoke.py:20
-#, fuzzy
 msgid "Filter _band:"
-msgstr "Filtrer sur ce _genre"
+msgstr "Filtrer sur le groupe"
 
 #: ../quodlibet/ext/gstreamer/karaoke.py:21
 msgid "The Frequency band of the filter"
 msgstr ""
 
 #: ../quodlibet/ext/gstreamer/karaoke.py:22
-#, fuzzy
 msgid "Filter _width:"
-msgstr "Filtrer sur cet _artiste"
+msgstr "Filtrer sur la taille"
```

Comment:
Same, "The Frequency width of the filter",  so "Largeur de bande du filtre" or "Largeur de bande"